<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="button_open">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>740</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_objects">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Lista de Objetos</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_viewport">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>40</y>
     <width>921</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Viewport</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_close">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>740</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphics_view_viewport">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>60</y>
     <width>931</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="list_objects">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>301</width>
     <height>491</height>
    </rect>
   </property>
   <property name="currentRow">
    <number>-1</number>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
